README.md

```diff
@@ -1,7 +1,5 @@
 # Platform Tools
 
-![](https://codebuild.eu-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiS2t1L3UvQmVTbXZsOTVIOWxGanpwTTh4b3BNcUR4c0dNN2NoSUpGcVkzN0JEOFpvc2kwL2pGVC91TXNVcjFNK0d5eExia0R2SS9lZUhuWTZQOTlieVY0PSIsIml2UGFyYW1ldGVyU3BlYyI6Im5tS0pUVEwvT204WXdxT2wiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)
```

Comment:
Why are we getting rid of this

Reply:
Sorry, I meant to put a comment there.

It's not telling us anything useful...
<img width="507" alt="image" src="https://github.com/user-attachments/assets/9643104e-7879-4c6e-97be-70f9b3700db5">